src/app/pages/MediaArticlePage/MediaArticlePage.styles.ts

```diff
@@ -27,6 +27,14 @@ export default {
         gridColumn: '1 / span 8',
       },
     }),
+  cafPrimaryColumn: ({ mq }: Theme) =>
+    css({
+      gridColumn: '1 / span 12',
+      paddingBottom: '2rem',
+      [mq.GROUP_4_MIN_WIDTH]: {
+        gridColumn: '3 / span 8',
+      },
+    }),
   secondaryColumn: ({ mq }: Theme) =>
```

Comment:
This centers the content succesfully when theres only one column, dont know if we have any assets where the secondary column is displayed but im guessing this actually wont work if thats rendered?  

Thinking maybe could just do a check for `isCaf`, and if its true just dont render the secondary column at all? As these pages will be low traffic anyway, does this sound okay to do ?

Reply:
For now I think we want the layout to be the same as the current MediaAssetPages, so no secondary column at all. We don't need the most watched section that appears below MediaAssetPages, so I feel like not showing secondary content on MAP pages in the MediaArticlePage component is fine.